packages/mux-player/src/template.ts

```diff
@@ -29,6 +29,7 @@ const isLiveOrDVR = (props: MuxTemplateProps) =>
 export const content = (props: MuxTemplateProps) => html`
   <${unsafeStatic(castThemeName(props.theme) ?? 'media-theme-mux')}
     has-src="${props.hasSrc}"
+    has-open-dialog="${props.isDialogOpen}"
```

Comment:
thought(non-blocking): We'll want to be thinking about how to handle these scenarios in a theme-agnostic way going forward.

Reply:
Yeah, as I mentioned, https://github.com/muxinc/elements/pull/427 is theoretically a more correct fix and would be theme agnostic. However, that change is a bit more complicated. There's potentially a performance issue but there's also an issue around binding the key against the document and trying to find all focusable elements and mux-player, which might not be possible depending on how mux-player is included on the page.